lib/mix/lib/mix/cli.ex

```diff
@@ -126,4 +133,13 @@ defmodule Mix.CLI do
       ["--version", "-v"], do: :version
 
   defp check_for_shortcuts(_), do: nil
+
+  defp usage() do
```

Comment:
Sorry @ktec I didn't mean to interfere. From what I was able to see my suggestion doesn't overlap this PR, so please go ahead with it.